website/docs/tutorials/basic/running_your_app/2_multirun.md

```diff
@@ -42,8 +42,9 @@ Additional sweepers are available as plugins.
 For example, the [Ax Sweeper](/plugins/ax_sweeper.md) can automatically find the best parameter combination!
 
 ### Launcher
-A Launcher is what runs your job, Hydra comes with a simple launcher that runs the jobs locally and serially.
-However, other launchers are available as plugins. For example - The [JobLib Launcher](/plugins/joblib_launcher.md)
-can execute the different parameter combinations in parallel on your local machine using multi-processing.
+A Launcher is what runs your job. Hydra comes with a simple launcher that runs the jobs locally and serially.
+However, other launchers are available as plugins. For example, the [JobLib Launcher](/plugins/joblib_launcher.md)
+can execute the different parameter combinations in parallel on your local machine using multi-processing, the 
+[Ray Launcher](/plugins/ray_launcher.md) can launch your application code to AWS, and the
+[Submitit Launcher](/plugins/submitit_launcher.md) enables launching to slurm.
```

Comment:
I think we don't need to mention all supported launchers here.
just to say that other launchers are available that can launch in parallel and to different clusters and point to the joblib as a representative one.

Reply:
ok :)